### PR TITLE
Make md and lg views of hero responsive

### PR DESCRIPTION
**Pull request by elliezub**
This fixes issue #10 

Sorry for the delay, I've been sick for a few days now 😢. 

I ended up editing the layout for md and lg, here's how it looks on my computer:

md:
md-deals

lg:
lg-deals

Please let me know if that looks ok to you or if I should change anything else, I also noticed on the smallest view the <p> text seems a bit long since it is much wider than the email input, so if you want me to I could add a max width? 
sm-deals

### CODE REVIEW

**Comment by chris-nowicki** [comment]
@elliezub this is already looking so much better!! 🤩

A few thoughts:

1.  Yes, on the small one I think we need to wrap that text that extends the form input.
2. Just wondering, did you try and center the text/form with the "Built With" text in the footer.  See reference image below.  Maybe try centering red box with purple line? Keep all the text alignment as is, but just shift everything over to the right?

CleanShot 2024-03-05 at 23 40 47

**Comment by elliezub** [comment]
Hey @chris-nowicki did you want it centered like that on all views equal or smaller than lg? I saw on the phone/tablet size views it was off to the left originally so I thought that's how it should be. Or do you want to keep it off to the left on the smallest screen size? or just center it for every size?

**Comment by chris-nowicki** [comment]
> Hey @chris-nowicki did you want it centered like that on all views equal or smaller than lg? I saw on the phone/tablet size views it was off to the left originally so I thought that's how it should be. Or do you want to keep it off to the left on the smallest screen size? or just center it for every size?

Yup! Let's see what it looks like centered.  I think ti will look nice!

**Comment by elliezub** [comment]
> > Hey @chris-nowicki did you want it centered like that on all views equal or smaller than lg? I saw on the phone/tablet size views it was off to the left originally so I thought that's how it should be. Or do you want to keep it off to the left on the smallest screen size? or just center it for every size?
> 
> Yup! Let's see what it looks like centered. I think ti will look nice!

Here's how it looks centered, I had to add some padding on the left so it looked like it was actually in the center, otherwise it looks a bit off 🥲 If you know of a better way to do it definitely lmk and I can change it. This was just the best solution I could come up with.
Screenshot 2024-03-09 230057

Text wrapped on sm view:
Screenshot 2024-03-09 230154

**Review by jamesqquick — APPROVED** [review]

File: src/components/Hero.tsx [hunk 1]
```diff
@@ -3,15 +3,15 @@ import SubscribeForm from './forms/SubscribeForm'
...
+      <div className="flex flex-col gap-6 md:gap-12 md:pb-32 lg:gap-16 xl:flex-row">
+        <h1 className="max-w-80 text-4xl font-semibold leading-10 tracking-tight text-white md:max-w-[650px] md:text-6xl md:leading-[67px] lg:text-7xl xl:max-w-[861px]">
           The best <span className="text-teal-500">deals</span> and{' '}
           <span className="text-teal-500">giveaways</span> for developers
         </h1>
 
         {/* subscription form */}
         <div>
-          <p className="mb-3 text-sm leading-5 text-white md:mb-6 md:text-2xl md:leading-8">
+          <p className="mb-3 text-sm leading-5 text-white md:mb-6 md:max-w-[550px] md:text-xl md:leading-8 lg:text-2xl">
```
[the remark this answers is not in the document]
Comment (jamesqquick) [reply src/components/Hero.tsx]: @all-contributors please add @elliezub for code

**Comment by CBID2** [comment]
@all-contributors please add @elliezub for code

**Comment by CBID2** [comment]
@all-contributors please add @elliezub for code

**Comment by CBID2** [comment]
@all-contributors please add @elliezub for code

**Pull request merged**